### ISSUE TITLE
Drop down elements are not getting visual focus / Keyboard focus except the element already in selected in "Automatically check for updates" & "Update channel" dropdown while navigating within the dropdown in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#350]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.

1. Launch Xamarin Workbooks application.
2. Navigate to Tools and Preferences. Invoke preferences tab. It will open the general tab of preferences. Invoke the update tab item using mouse (As tab items are not accessible by keyboard)
3. Navigate "Automatically check for updates" & update channel dropdowns and navigate among the dropdown elements.
### Expected Behavior
Visual focus/Keyboard focus should appear on dropdown elements upon getting the focus
### Actual Behavior
Except the element in selected state, rest of the elements in the dropdown are not getting the visual keyboard focus around them in high contrast black mode.
### User Impact
User will not know whether the element in the dopdown got the focus or not.
### Screenshots
![10 hc black no focus in dropdown](https://user-images.githubusercontent.com/22956130/40723543-beb88fbe-643c-11e8-9bd6-c02b8b06fc7b.png)
![10 hc white with focus](https://user-images.githubusercontent.com/22956130/40723544-bef27f94-643c-11e8-90b4-eaf4bdfde989.png)
![10 normal mode](https://user-images.githubusercontent.com/22956130/40723545-bf259190-643c-11e8-9ddd-db4c72e04302.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
